tests/test_expressions.py

```diff
@@ -43,6 +43,9 @@ def test_find_all(self):
     def test_sql(self):
         assert parse('x + y * 2')[0].sql() == 'x + y * 2'
         assert parse('select "x"')[0].sql(dialect='hive', pretty=True) == 'SELECT\n  `x`'
+        assert parse('update tbl_name set foo = 123')[0].sql() == 'UPDATE tbl_name SET foo = 123'
```

Comment:
can you delete these tests and instead add them to identity?

also, add a test case for set x = 1, y =2

Reply:
Thanks, I've moved the test cases to identity